src/screen/loading.rs

```diff
@@ -0,0 +1,44 @@
+//! A credits screen that can be accessed from the title screen.
```

Comment:
I think you forgot to update this comment ;)

Reply:
😭